packages/fetcher/src/config.rs

```diff
@@ -1,6 +1,25 @@
 use bdk::prelude::*;
 use by_types::config::*;
 
+#[derive(Debug)]
+pub struct AwsConfig {
+    pub region: &'static str,
+    pub access_key_id: &'static str,
+    pub secret_access_key: &'static str,
+}
+
+impl Default for AwsConfig {
+    fn default() -> Self {
+        AwsConfig {
+            region: option_env!("AWS_REGION").expect("You must set AWS_REGION"),
+            access_key_id: option_env!("AWS_ACCESS_KEY_ID")
+                .expect("You must set AWS_ACCESS_KEY_ID"),
+            secret_access_key: option_env!("AWS_SECRET_ACCESS_KEY")
+                .expect("AWS_SECRET_ACCESS_KEY is required"),
+        }
+    }
+}
```

Comment:
```suggestion
}
```

[//packages/main-api/src/config.rs
](https://github.com/biyard/ratel/blob/dev/packages/main-api/src/config.rs)
이 코드 처럼 그냥 `aws: AwsConfig`, 만 넣어도 동작합니다.
`by_types` 에  default 동ㅈ가이 지정되어 있어서요.